docs/html/reference/pip_install.rst

```diff
@@ -846,7 +846,6 @@ Examples
       $ pip install SomePackage[PDF]
       $ pip install git+https://git.repo/some_pkg.git#egg=SomePackage[PDF]
       $ pip install SomePackage[PDF]==3.0
-      $ pip install -e .[PDF]==3.0  # editable project in current directory
```

Comment:
How about changing the example to `$ pip install -e .[PDF]  # editable project in current directory` instead?

Reply:
Thanks for the hint.